jsk_mbzirc_common/gazebo_model/world/mbzirc_arena_task_2.world

```diff
@@ -31,7 +31,7 @@
     </include>
 
     <include>
-      <pose>0 0 0 1.57 -0 0</pose>
+<!--      <pose>0 0 0 1.57 -0 0</pose> -->
```

Comment:
please remove line, instead of comment, if you have a reason, please add comment to why you comment out this line.